core/src/prediction/LLCausalPredictionStrategy.cpp

```diff
@@ -147,8 +150,152 @@ std::vector<double> LLCausalPredictionStrategy::compute_variance(
         size_t sampleID,
         std::vector<std::vector<size_t>> samples_by_tree,
         std::unordered_map<size_t, double> weights_by_sampleID,
-        const Data* train_data,
-        const Data* data,
+        const Data* original_data,
```

Comment:
Small comment, maybe we could stick with `train_data` and `data`, since that's what we use in `LocalLinearPredictionStrategy` (and fix the naming in the other parts of this file)?